components/Organizer/BugReportsTab/BugReportsTab.tsx

```diff
@@ -0,0 +1,314 @@
+import { Button, Input, InputRef, Skeleton, Space, Table, Tag } from 'antd';
+import { RequestType, useCustomSWR } from '../../../utils/request-utils';
+import { GitHubIssueStatus, Report } from '../../../types/client';
+import { ColumnsType } from 'antd/lib/table';
+import { DeleteOutlined, SearchOutlined } from '@ant-design/icons';
+import { FilterConfirmProps } from 'antd/es/table/interface';
+import Highlighter from 'react-highlight-words';
+import { useEffect, useRef, useState } from 'react';
+import { FilterValue } from 'antd/lib/table/interface';
+import styles from '../../../styles/Report.module.css';
+import { Octokit } from 'octokit';
+
+const BugReportsTab = () => {
+	const searchInput = useRef<InputRef>(null);
+	const [searchText, setSearchText] = useState('');
+	const [searchedColumn, setSearchedColumn] = useState('');
+	const [filteredInfo, setFilteredInfo] = useState<Record<string, FilterValue | null>>({
+		status: [],
+	});
+	const [githubIssues, setGithubIssues] = useState<GitHubIssueStatus[] | null>(null);
+	const [githubIssueLoading, setGithubIssueLoading] = useState<boolean>(false);
+
+	const { data: bugReports, error: bugReportsError } = useCustomSWR<Report[]>({
+		url: '/api/report',
+		method: RequestType.GET,
+		errorMessage: 'Failed to get bug reports.',
+	});
```

Comment:
`useSWR` allows us to pass in an extra JSON, for example: 
```ts
const { data: user } = useSWR(
		'/api/user-data',
		async url => {
			const res = await fetch(url, { method: 'GET' });
			return (await res.json()) as UserData;
		},
		{ revalidateOnFocus: false, revalidateOnMount: true }
	);
```

perhaps a future TODO for us to refactor useCustomSWR to be able to pass that. 

well, how about i create an issue using this PR? 🤩 [https://github.com/VandyHacks/witness/issues/410](https://github.com/VandyHacks/witness/issues/410)